src/sandstorm/spk.c++

```diff
@@ -889,11 +889,99 @@ private:
         "      )\n"
         "    ],\n"
         "\n"
-        "    continueCommand = .myCommand\n"
+        "    continueCommand = .myCommand,\n"
         "    # This is the command called to start your app back up after it has been\n"
         "    # shut down for inactivity. Here we're using the same command as for\n"
         "    # starting a new instance, but you could use different commands for each\n"
         "    # case.\n"
+        "\n"
+        "    metadata = (\n"
+        "      # Data which is not needed specifically to execute the app, but is useful\n"
+        "      # for purposes like marketing and display.  These fields are documented at\n"
+        "      # https://docs.sandstorm.io/en/latest/developing/publishing-apps/#add-required-metadata\n"
+        "      # and in the source code in the Metadata section of\n"
+        "      # https://github.com/sandstorm-io/sandstorm/blob/master/src/sandstorm/package.capnp\n"
```

Comment:
-0 on source code link here; the docs should be enough and if they're not we should fix that IMHO (but if you think it's essential I can live)